improver/calibration/rainforest_calibration.py

```diff
@@ -85,13 +85,13 @@ def __init__(self, model_config_dict: dict, threads: int):
 
         try:
             from treelite_runtime import Predictor
-
-            self.treelite_enabled = True
         except ModuleNotFoundError:
             warnings.warn(
                 "Module treelite_runtime unavailable. Defaulting to using lightgbm Boosters."
             )
             self.treelite_enabled = False
+        else:
+            self.treelite_enabled = True
```

Comment:
We could also do the same with the import in `test_ApplyRainForestsCalibration.py`.